sci-electronics/kicad/kicad-4.0.0_rc1.ebuild

```diff
@@ -0,0 +1,178 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit bzr cmake-utils fdo-mime flag-o-matic gnome2-utils python-r1 vcs-snapshot wxwidgets versionator
```

Comment:
bzr is needed to get patched boost from repo if bundled boost is used instead of system one. I agree however it should be conditional on "!boost".